provider/ansible/documentation.rb

```diff
@@ -112,7 +112,7 @@ def format_description(desc)
         desc.split(".\n").map do |paragraph|
           paragraph += '.' unless paragraph.end_with?('.')
           paragraph = format_url(paragraph)
-          paragraph.tr("\n", ' ').strip.squeeze(' ')
+          paragraph.tr("\n", ' ').strip.squeeze(' ').gsub('`', "'")
```

Comment:
It looks like somehow some of the generated files are showing ''s in places (but not everywhere). any idea what's going on?